entity-management-zoho/src/main/java/eu/europeana/entitymanagement/zoho/organization/ZohoOrganizationConverter.java

```diff
@@ -85,6 +85,10 @@ public static Organization convertToOrganizationEntity(Record zohoRecord) {
       org.setAddress(address);
     }
 
+    List<String> edmLanguage =
+        ZohoUtils.stringListSupplier(zohoRecord.getKeyValue(ZohoConstants.OFFICIAL_LANGUAGE_FIELD));
+    if (!edmLanguage.isEmpty()) org.setEdmLanguage(edmLanguage);
```

Comment:
please use the {} for the body of the if statement, as it breaked the sonar cube rules

Reply:
Done.